src/api/app/models/bs_request/find_for/base.rb

```diff
@@ -45,7 +45,8 @@ def types
       end
 
       def review_states
-        @parameters[:review_states] || [:new]
+        result = [@parameters[:review_states]].flatten.compact
```

Comment:
Why is `.flatten.compact` necessary?

Reply:
Because I wrap it into another array to make it possible to call ``BsRequest::FindFor.new(review_states: :new)`` or ``BsRequest::FindFor.new(review_states: [:new])``. So in latter case it would produce ``[[:new]]`` (flatten). In case the review_states is nil it will produce ``[nil]`` which would not be empty (next line).